presto-spi/pom.xml

```diff
@@ -28,6 +28,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
```

Comment:
I'm concerned about adding this to the SPI, as it could have versioning issues for plugins, since it changes frequently.